### PR TITLE
AC-695 Added Unknown Patient Checkbox

### PR DESCRIPTION
**Pull request by rishabh-997**
## Description of what I changed
1. Added checkbox for unidentified patients
2. Hide different blocks base on checkbox listener
3. Create unknown patient with gender and estimated DOB
4. Not checking similar patients in case of unidentified patients 

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-695

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by rishabh-997** [comment]
I'll convert layout into constraint layout once I design a UI for it. Same for view binding.

**Comment by rishabh-997** [comment]
[IMAGE ALT TEXT HERE](https://firebasestorage.googleapis.com/v0/b/real-time-tracking-89266.appspot.com/o/whatsapp-video-2020-05-11-at-24555-pm-1_6PA4MHjJ_LOv1.mp4?alt=media&token=2fa9b809-6238-44a4-ba7e-9e77ff22d142)

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/addeditpatient/AddEditPatientFragment.java [hunk 1]
```diff
@@ -112,6 +118,12 @@
     private LocalDate birthdate;
     private DateTime bdt;
 
+    private LinearLayout linearLayout_name;
+    private RelativeLayout relativeLayout_dob;
+    private LinearLayout linearLayout_contact_info;
+    private CheckBox unidentified_checkBox;
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/addeditpatient/AddEditPatientFragment.java:124]: Please use `camelCase` for the variable names. Should be `checkBoxUnidentified` to match the pattern.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/addeditpatient/AddEditPatientFragment.java [hunk 2]
```diff
@@ -225,104 +240,109 @@ public void scrollToTop() {
...
-            firstNameTIL.setError(emptyError);
-        } else if (!ViewUtils.validateText(ViewUtils.getInput(edfName), ViewUtils.ILLEGAL_CHARACTERS)) {
-            lNameError.setText(familyNameError);
-            firstNameTIL.setErrorEnabled(true);
-            firstNameTIL.setError(givenNameError);
-        } else {
-            firstNameTIL.setErrorEnabled(false);
-        }
+            PersonName name = new PersonName();
+            name.setFamilyName("UNKNOWN");
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/addeditpatient/AddEditPatientFragment.java:247]: Place this string in `ApplicationConstants`

**Pull request merged**